great_expectations/rule_based_profiler/domain_builder/column_pair_domain_builder.py

```diff
@@ -26,6 +26,16 @@ class ColumnPairDomainBuilder(ColumnDomainBuilder):
     This DomainBuilder uses relative tolerance of specified map metric to identify domains.
     """
 
+    exclude_field_names: Set[str] = ColumnDomainBuilder.exclude_field_names | {
+        "exclude_column_names",
+        "include_column_name_suffixes",
+        "exclude_column_name_suffixes",
+        "semantic_type_filter_module_name",
+        "semantic_type_filter_class_name",
+        "include_semantic_types",
+        "exclude_semantic_types",
+    }
```

Comment:
How can we make these strings less "magical"? Can they be converted to a `StrEnum`?

Reply:
@cdkini I am not sure -- we talked about this before; I think that a redesign of `IDDict` is necessary to improve (hence, we had left it for later).  Thanks.

Reply:
sure thing